rabbitmq/README.md

```diff
@@ -38,7 +36,10 @@ Configure the `prometheus_plugin` section in your instance configuration. When u
        url: http://<HOST>:15692
  ```
 
- This enables scraping of the [`/metrics` endpoint][20] on one rabbitmq node.
+ This enables scraping of the [`/metrics` endpoint][20] on one rabbitmq node. We can also collect data from the [`/metrics/detailed` endpoint][22].
```

Comment:
```suggestion
 This enables scraping of the [`/metrics` endpoint][20] on one RabbitMQ node. We can also collect data from the [`/metrics/detailed` endpoint][22].
```
Tiny nit

Reply:
Thanks! I went through the whole file and fixed up some more inconsistent namings!